editor/src/clj/editor/connection_properties.clj

```diff
@@ -12,14 +12,14 @@
 ;; CONDITIONS OF ANY KIND, either express or implied. See the License for the
 ;; specific language governing permissions and limitations under the License.
 
-(ns editor.connection-properties)
-
-(def ^:const connection-properties
-  {:sentry {:project-id "97739"
-            :key        "9e25fea9bc334227b588829dd60265c1"
-            :secret     "f694ef98d47d42cf8bb67ef18a4e9cdb"}
-   :google-analytics {:tid "UA-83690-7"}
-   :git-issues {:url "https://github.com/defold/defold"}
-   :native-extensions {:build-server-url "https://build.defold.com"}
-   :updater {:download-url-template "https://%s/archive/%s/%s/editor2/Defold-%s.zip"
-             :update-url-template "https://%s/editor2/channels/%s/update-v3.json"}})
+(ns editor.connection-properties)
+
+(def ^:const connection-properties
+  {:sentry {:project-id "97739"
+            :key "9e25fea9bc334227b588829dd60265c1"
+            :secret "f694ef98d47d42cf8bb67ef18a4e9cdb"}
+   :google-analytics {:tid "UA-83690-7"}
+   :git-issues {:url "https://github.com/defold/defold"}
+   :native-extensions {:build-server-url "https://build.defold.com"}
+   :updater {:download-url-template "https://%s/archive/%s/%s/editor2/Defold-%s.zip"
+             :update-url-template "https://%s/editor2/channels/%s/update-v4.json"}})
```

Comment:
This file had mixed line endings, but the only significant change is that we now look for updates in `update-v4.json` instead of `update-v3.json`.

Reply:
Where do we create this update-v4.json?